backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/internal/BpkTextFieldImpl.kt

```diff
@@ -333,12 +333,13 @@ private fun RowScope.TrailingIcon(
 internal enum class BpkTextFieldType {
     Default,
     Search,
+    Select,
```

Comment:
So if that's the case, anyone can create a TextField with Select appearance even though it's only supposed to be used internally. If it's not a part of the API design we have in Figma, it should be done via internal mappings instead.

Reply:
`BpkTextFieldType` is only used in `BpkTextFieldImpl` so not available to consumers was my understanding